build-system/tasks/runtime-test.js

```diff
@@ -73,21 +73,20 @@ function getConfig() {
       reporters: ['super-dots', 'saucelabs', 'karmaSimpleReporter'],
       browsers: argv.saucelabs ? [
         // With --saucelabs, integration tests are run on this set of browsers.
-        'SL_Chrome_67',
-        'SL_Firefox_61',
-        'SL_Safari_11',
-        // TODO(rsimha, #16687): Enable after Sauce disconnects are resolved.
-        // 'SL_Chrome_Android_7',
-        // 'SL_Chrome_45',
-        // 'SL_Android_6',
-        // 'SL_iOS_11',
-        // 'SL_Edge_17',
-        // 'SL_IE_11',
+        'SL_Chrome',
+        'SL_Chrome_Dev',
+        'SL_Firefox',
+        'SL_Safari_12',
+        'SL_Chrome_Android_7',
+        'SL_iOS_12',
+        'SL_Edge_17',
+        'SL_IE_11',
```

Comment:
I suspect the proxy will get overloaded if we add so many platforms in one go. An alternative is what we talked about yesterday: split into two sets and run one after another as two consecutive integration test invocations.

Reply:
IIRC, `concurrency` only controls the number of platforms on which tests run in parallel. It doesn't restrict the number of tunnels created. Worth experimenting with, though.

Reply:
Yup you're right. Batched the karma test instead with `const batchSize = 4;` which seems to work okay with the SauceLabs tunnels.